instaloader/instaloader.py

```diff
@@ -1605,3 +1615,9 @@ def interactive_login(self, username: str) -> None:
                 except BadCredentialsException as err:
                     print(err, file=sys.stderr)
                     pass
+
+    def has_stored_errors(self) -> bool:
```

Comment:
I think this should be a property (and the method in InstaloaderContext as well)

```suggestion
    @property
    def has_stored_errors(self) -> bool:
```